src/main/java/core/basesyntax/service/FruitShop.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitsTransaction;
+import core.basesyntax.strategy.Strategy;
+import java.util.List;
+
+public class FruitShop {
+    private final Strategy strategy;
+
+    public FruitShop(Strategy strategy) {
+        this.strategy = strategy;
+    }
+
+    public void processOfOperations(List<FruitsTransaction> operationList) {
+        for (FruitsTransaction operation : operationList) {
```

Comment:
java is strongly typed, we don't usually include type in the variable name
```suggestion
    public void processOfOperations(List<FruitsTransaction> operations) {
        for (FruitsTransaction operation : operations) {
```